gio/src/subclass/input_stream.rs

```diff
@@ -363,10 +363,7 @@ mod tests {
 
     #[test]
     fn test_simple_stream() {
-        let stream = glib::Object::new(SimpleInputStream::static_type(), &[])
-            .unwrap()
-            .downcast::<SimpleInputStream>()
-            .unwrap();
+        let stream: SimpleInputStream = glib::Object::new(&[]).unwrap();
```

Comment:
```suggestion
        let stream = glib::Object::new::<SimpleInputStream>(&[]).unwrap();
```

And elsewhere

Reply:
I personally prefer this style over the turbofish, but I can change it.

Reply:
I think that's what we're using everywhere else so for consistency at least...